src/streamlink_cli/output.py

```diff
@@ -160,7 +162,15 @@ def _close(self):
 
         if self.kill:
             with ignored(Exception):
-                self.player.kill()
+                self.player.terminate()
+                if not is_win32:
+                    t, timeout = 0.0, self.PLAYER_TERMINATE_TIMEOUT
+                    while not self.player.poll() and t < timeout:
```

Comment:
Shouldn't this be `while self.player.poll() is None` ?
Now it waits if the player returned 0, for example if I close mpv by pressing Q...